tests/ruby/tree-sitter-ruby.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+/*
+  Parse a source file with a tree-sitter-generated parser and dump
+  the concrete syntax tree (CST) in json.
+
+  Warning: This file is meant to be copied into the current folder.
+  A direct call or a symbolic link won't work due to "require('.')".
+
+  Requires:
+  - tree-sitter, tree-sitter-cli.
+  - the products of running 'tree-sitter generate' in the current folder.
+*/
+
+const Parser = require('tree-sitter');
+const pl = require('tree-sitter-ruby');
```

Comment:
I copy-pasted the whole file and this is the only line that changed. Will work on packaging parsers cleanly later.

Reply:
yep, copy-paste is fine for fast prototyping :)